### DISCUSSION TITLE
Présentation du projet

### DISCUSSION THREAD

**Discussion by Anatharr** [Campus-INSA-CVL/campus-website#110] [Team Posts]
# Introduction

**Bonjour à tous !**
Comme vous vous en doutez, le but de ce groupe va être de faire la refonte du site campus, qui en a bien besoin.

Pour l'instant @Barbapapazes s'est déjà lancé (bravo à toi) et nous a fait un [notion.so](https://www.notion.so/Site-Campus-efbae143c5f84d9c9d1861c5be16cab0) très détaillé, que je vous encourage vivement à lire.

# Technologie

La principale contrainte est le fait de devoir **prévoir la passation**. Les prochaines personnes qui seront amenées à maintenir ce site n'auront pas nécessairement de connaissances en web development, et il faut qu'elles soient en mesure d'ajouter et d'éditer du contenu facilement. Pour cela nous allons utiliser le framework **[NuxtJS](https://nuxtjs.org/)**, qui nous permettra de ne pas créer d'API, car tous les contenus seront directement ajoutés dans les sources, avec une syntaxe simple de markdown (petit exemple ci-dessous), d'où l'utilisation de GitHub. Tout cela est mieux expliqué dans le lien de @Barbapapazes qui a trouvé cette solution donc une fois de plus allez le voir. 

![Exemple de markdown](https://user-images.githubusercontent.com/45267552/86158190-b8a25400-bb08-11ea-9499-f3d2bf4b7b17.png)

# Organisation

Sachant qu'il y aura aussi l'application et l'api à maintenir, dites moi ce qui vous intéresserait le plus à faire pour qu'on s'organise, que je vous ajoute dans les sous groupes correspondants, et qu'on puisse commencer à faire les todo lists.

_Cheers._

**Comment by Anatharr** [comment]
Du coup la création d'un groupe messenger n'est pas forcément nécessaire, on peut communiquer directement ici

**Comment by Barbapapazes** [comment]
tu ferras gaffe mais ce que tu nommes comme étant l'exemple ci-dessous est du yaml. Je te conseil de mettre ça plutôt !
![image](https://user-images.githubusercontent.com/45267552/86158190-b8a25400-bb08-11ea-9499-f3d2bf4b7b17.png)

**Comment by Anatharr** [comment]
Ah my bad le yaml c'est pas un type de md ?

**Comment by Barbapapazes** [comment]
> Ah my bad le yaml c'est pas un type de md ?

Non, le yaml c'est des fichiers de structures de data, comme le json ou le xml alors que le md c'est pour la sémantique d'un texte, comme peut l'être le HTML

**Comment by Anatharr** [comment]
Ca marche, c'est modifié :+1:
